index.js

```diff
@@ -1,2 +1,94 @@
+// Load environment variables
+require('dotenv').config();
+
+const slackEventsApi = require('@slack/events-api');
+const passport = require('passport');
+const SlackStrategy = require('@aoberoi/passport-slack').default.Strategy;
+const http = require('http');
 const express = require('express');
-const slack = require('slack');
\ No newline at end of file
+const bodyParser = require('body-parser');
+
+const { getClientByTeamId } = require('./util/getClient');
+
+const port = process.env.APP_PORT || 3000;
+
+// Initialize a data structures to store team authorization info (typically stored in a database)
+const botAuthorizations = {}
+
+// *** Initialize event adapter using verification token from environment variables ***
+const slackEvents = slackEventsApi.createSlackEventAdapter(
+  process.env.SLACK_VERIFICATION_TOKEN, 
+  { includeBody: true }
+);
+
+// Initialize Add to Slack (OAuth) helpers
+passport.use(new SlackStrategy({
```

Comment:
Just a side note - lets move this all to an authentication.js file in another pR